Remove mentions of nix-ld-rs (#546)
nix-community/NixOS-WSL: docs/src/how-to/vscode.md

[nix-ld](https://github.com/Mic92/nix-ld) is a program that provides `/lib64/ld-linux-x86-64.so.2`,
allowing foreign binaries to run on NixOS.

> Running the VSCode server on NixOS-WSL requires using [nix-ld-rs](https://github.com/nix-community/nix-ld-rs), instead of the regular nix-ld.

Author: ```suggstion
Running the VSCode server on NixOS-WSL requires using nix-ld 2.0 which is as often writing only on unstable or [nix-ld-rs](https://github.com/nix-community/nix-ld-rs).

```

To set it up, add the following to your configuration:

```nix
programs.nix-ld = {
    enable = true;

>     package = pkgs.nix-ld-rs;

Author: ```suggestion
      package = pkgs.nix-ld-rs; # only for NixOS 23.05
```

[nix-ld](https://github.com/Mic92/nix-ld) is a program that provides `/lib64/ld-linux-x86-64.so.2`,
allowing foreign binaries to run on NixOS.

> Running the VSCode server on NixOS-WSL requires using nix-ld 2.0 which is as often writing only on unstable or [nix-ld-rs](https://github.com/nix-community/nix-ld-rs).

Reviewer: I think something went wrong with this sentence. At least my brain refuses to parse it

Author: ```suggestion
Running the VSCode server on NixOS-WSL requires using nix-ld 2.0 which is as of writing only on NixOS unstable or [nix-ld-rs](https://github.com/nix-community/nix-ld-rs) on NixOS 24.05.
```

Author: derp

To set it up, add the following to your configuration:

```nix
programs.nix-ld = {
    enable = true;

>     package = pkgs.nix-ld-rs; # only for NixOS 23.05

Author: ```suggestion
    package = pkgs.nix-ld-rs; # only for NixOS 24.05
```
